imap/cyr_expire.c

```diff
@@ -624,7 +626,7 @@ static int expire_conversations(const mbentry_t *mbentry, void *rock)
 
 static void sighandler(int sig __attribute((unused)))
 {
-    sigquit = 1;
+    in_shutdown = 1;
```

Comment:
If we start setting `in_shutdown` from a signal handler like this, then it needs to change type to `volatile sig_atomic_t`, like `sigquit` was.  Looks like `in_shutdown` is just an `int`, and comes from imap/global.c

Though... it feels wrong that `cyr_expire` even has its own signal handler.  Shouldn't it be using the lib/signals.c functions like everything else?

Reply:
yeah, it's probably old.  I'll change that.

(sig_atomic_t probably isn't needed given that it only ever goes 0 to 1 and it isn't the end of the world to miss the value temporarily, but also - may as well change the global to be that too)